rulesets/src/naming.ruleset.yml

```diff
@@ -1,5 +1,47 @@
 rules:
-  # domain references (URN-like values)
+  ##### General #####
+  sps-no-keyword-conflicts:
+    description: Names that may conflict with keywords in common programming languages SHOULD NOT be used.
+    severity: warn
+    given: "$..properties.*~"
```

Comment:
this will pick up all schemas... wondering if we want to catch query parameter names as well?

Reply:
None of the current rules apply to query parameters as far as I'm aware, however I believe it should be possible to update the json path to do so if we'd like

Reply:
Lets leave as is, impacting the schema names for now.